demo/mpfao.c

```diff
@@ -0,0 +1,289 @@
+#include "tdycore.h"
```

Comment:
I am wondering if we really need this file? As far as I can see you set it up to do what the steady.c does. Can we just use steady.c and then change the method with `-tdy_method {wy|bdm|tpf|mpfa_o}`?